### PR TITLE
fix: update proper error messages

### LINKED ISSUE #86: Update the missing arguments message
https://github.com/asyncapi/cli/blob/9d385168166957be89cbc52692b69017c07d1dda/src/messages.tsx#L8

It should say what arguments are missing, For example for the command `asyncapi context add <context-name> <spec-path>`
The message should be something like this. 
```
Missing arguments: context-name
run asyncapi <command> --help to know more. 
````

A couple of things needs to be changed - 
- https://github.com/asyncapi/cli/blob/9d385168166957be89cbc52692b69017c07d1dda/src/messages.tsx#L8 it has to be converted into a function that would accept `parameter_name` and `command`. 
- Then update the Error class https://github.com/asyncapi/cli/blob/9d385168166957be89cbc52692b69017c07d1dda/src/hooks/context/models.ts#L37-L42 
- and finally updated all the instances where this error has been thrown.

### PR DESCRIPTION
**Pull request by mihirterna**
**Description**

- There were missing arguments message, as per issue #86 
- Updated message files to contain expected error messages

**Related issue(s)**
Fixes #86

### CODE REVIEW

File: src/messages.tsx [hunk 1]
```diff
@@ -21,3 +21,9 @@ export const NO_SPEC_FOUND = (command: string) => `${FALLBACK_MESSAGES[command]}
 export const FALLBACK_MESSAGES: {[name: string]: string} = {
   validate: 'Unable to perform validation. Specify what AsyncAPI file should be validated.'
 };
+
+export const CONTEXT_NAME = 'context-name';
+
+export const SPEC_PATH = 'spec-path';
+
+export const CONTEXT = 'context';
```
Comment (Souvikns) [inline src/messages.tsx:29]:
```suggestion
export const Arguments = {
  context_name: 'context-name',
  spec_path: 'spec-path',
  context: 'context'
}
```

I think exporting an `Argument` object would be good as they will not clash with other variables and it is easy to understand what these constants are.

**Review by Souvikns — CHANGES_REQUESTED** [review]
I left some comments, and the rest is fine. Good job @mihirterna 👍🏻

File: src/messages.tsx (hunk 1 above)
Comment (magicmatatjahu) [reply src/messages.tsx:29]: Generally in JS global variables should be written with capital letters:

```js
export const ARGUMENTS = {
  CONTEXT_NAME: 'context-name',
  SPEC_PATH: 'spec-path',
  CONTEXT: 'context'
}
```

Comment (mihirterna) [reply src/messages.tsx]: Thanks for review, have updated above changes. Please check.

**Review by Souvikns — DISMISSED** [review]
LGTM 🚀. Just have to add the `hacktoberfest-accepted` label.

**Comment by magicmatatjahu** [comment]
@Souvikns `hacktoberfest-accepted` label can be skipped - repo has such an label to take participant in hacktoberfest :)

**Comment by magicmatatjahu** [comment]
@mihirterna Please run linter (https://github.com/asyncapi/cli/pull/87/checks?check_run_id=3791303609) and push changes - `npm run lint`.

**Comment by mihirterna** [comment]
> @mihirterna Please run linter (https://github.com/asyncapi/cli/pull/87/checks?check_run_id=3791303609) and push changes - `npm run lint`.

Thanks for comment, have committed formatted code.

**Review by Souvikns — APPROVED** [review]
LGTM, @magicmatatjahu shall I merge this?

**Comment by magicmatatjahu** [comment]
@Souvikns I had to change PR's title so I did it for you :)

**Pull request merged**